bench_test.go

```diff
@@ -11,32 +11,48 @@ import (
 
 func BenchmarkBase58EncodeToString(b *testing.B) {
 	data := make([]byte, 8192)
+	data[0] = 0xff // without this, it's just skipping zero bytes
```

Comment:
Also curious about the skipping.. what's the internal mechanism you're working around here?